app/icon.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
Sure, but I thought it would work well as a reusable component for small inline SVG shapes.